articles/types-of-styling-in-css/index.md

```diff
@@ -0,0 +1,150 @@
+
+However, this does not mean that the style should not be used. It usually works well when you're trying to test out a new style on an element making it easier to determine if the style works well with the design or not.
+
+Always keep in mind that local styles have a higher priority over page-level and external styles. Meaning they will override any style property defined within page-level and external.
+
+**Pros:**
+- Easy to test designs on individual elements.
+- You can use one document for the loading of CSS styles.
+
+**Cons:**
+- It can take a long time to load or render a page since multiple individual elements are styled within.
+- This final file will look disorganized, thus making it harder to read.
+
+- **Page Level Style** is the clean version local style where you have everything located in one central place within the header of the HTML file. This gives the best results especially when you're in the development phase of the project. This is because browsers do not cache page-level styles like they do external styles.
+
+**Pros:**
+- Easy to manage during initial development stages.
+- Programmer doesn't need to keep switching files to make minor adjustments.
+
+**Cons:**
+- Adding code to the document increases the size of the file.
+- In order to manipulate a website design you would have to acquire all the files that contain CSS.
+
+- **External Styles** gives you a wider range for its implementation. Since all the code used in the page styling is stored on a separate file it allows one to perform global changes on the project since one style sheet can control many pages.
+
+I prefer using external styling when it comes to launching the project or working with a live webpage. It is easier to upgrade since the design parameters of the entire site are defined in a single file.
+
+**Pros:**
+- One style sheet controls many pages.
+- Separation of content and design.
+
+**Cons:**
+- You can not render the page until the entire CSS file is downloaded.
+- Linking multiple CSS files to the page can lead to downtime.
+
+### Conclusion
+Whether it's testing out a new style on an element, building a simple single-page website, or building a company website that requires a uniform design from start to finish, you might dabble in inline CSS, page-level styles, or external styles respectively.
+
+Where inline lets you test out new design patterns but becomes too much of a workload implementing on the entire page. Page-level lets you play around with the entire web page design without leaving the page but proves disadvantageous when dealing with multiple pages. While external styles can give your website the feel of a whole new theme.
```

Comment:
These inline comments are just a few - to help you clarify your thinking and help polish the article.
Please go through the whole article as there may be many things I did not point out - but still may require attention.